pkg/kubectl/cmd/drain.go

```diff
@@ -464,9 +464,9 @@ func (o *DrainOptions) deleteOrEvictPods(pods []api.Pod) error {
 
 	if len(policyGroupVersion) > 0 {
 		return o.evictPods(pods, policyGroupVersion, getPodFn)
-	} else {
-		return o.deletePods(pods, getPodFn)
 	}
+	return o.deletePods(pods, getPodFn)
```

Comment:
I'm sorry, but could you please leave this and below as it is? I'm afraid the modification you made here makes the logic non-readable.